ssl/t1_trce.c

```diff
@@ -883,38 +883,43 @@ static int ssl_print_extensions(BIO *bio, int indent, int server,
     size_t extslen, msglen = *msginlen;
     const unsigned char *msg = *msgin;
 
-    BIO_indent(bio, indent, 80);
     if (msglen == 0) {
-        BIO_puts(bio, "No Extensions\n");
```

Comment:
Why are you removing this?

Reply:
at first glance, seemed rather superfluos output with not much informational value. After
reading up in more detail on TLS packet encoding, I realized this case would only apply
to older TLS/SSL versions before extensions were introduced. Dont remember when that happened. For modern versions, we will instead hit the extslen == 0 case. I'm not opposed to keeping the No extensions printing, but should probably also add it to the extslen == 0 case
but feel it doesnt add much value ?

Reply:
It's there for the case of an SSLv3 ClientHello which does not have an extensions block.

Reply:
(so I think it should be put back)